package.json

```diff
@@ -60,6 +60,7 @@
     "@types/node": "^12.12.24",
     "@types/react": "^16.9.17",
     "@types/react-paginate": "^6.2.1",
+    "autoprefixer": "^9.7.4",
```

Comment:
Why do we need it? Isn't enough the one from tailwindcss?

Reply:
It is needed for the browser-specific prefixes such as `-moz`. We have the same in listings too. Maybe we can drop it in the future since we already dropped IE support

Reply:
I know what it does, but is already required by tailwindCSS, therefore, it should be already installed, I think there is no need to install it separately, can you please check?

Reply:
I tested it, and the datepicker still looks fine. I'm not sure though if it affects anything else.
Based on this section, it only has the autoprefixer built-in if you use `postcss-preset-env`
https://tailwindcss.com/docs/using-with-preprocessors#future-css-features

I can remove it and see what happens?

Reply:
sure, go ahead. otherwise. you can also always give it to QA for cross-browser testing

Reply:
Yes correct, we don't require postcss-preset-env but we only require and use the autoprefixer that's why it works and there is no need to install it separately.
We can also replace `require("autoprefixer") with require("postcss-preset-env")` so we will have a better support

Reply:
for some reason, I have to install it manually with this change. It removes the autoprefixer package from the node_modules otherwise (tested on my local and in the pipeline)